labelit/mainapp/views.py

```diff
@@ -32,7 +32,7 @@ def dispatch(self, request, *args, **kwargs):
         logger.debug(f"Calling {self._upstream}")
         return super().dispatch(request, *args, **kwargs)
 
-@login_required
+@login_required(login_url='/login')
```

Comment:
There's a `LOGIN_URL` setting in settings.py. Modify that to `/login` instead of giving it explicitly here